src/main/java/com/squareup/catalog/demo/Main.java

```diff
@@ -128,7 +128,9 @@ void processArgs(String[] args) {
     }
 
     // Process arguments associated with the example.
-    String accessToken = null;
+    // By default, set the value of the access token to the environment variable SQPAT,
+    // as mentioned in the README.
+    String accessToken = System.getenv("SQPAT");
```

Comment:
let's change this envar to something more understandable like: `SQUARE_ACCESS_TOKEN`